src/ng/directive/select.js

```diff
@@ -98,6 +109,39 @@ var SelectController =
   self.hasOption = function(value) {
     return !!optionsMap.get(value);
   };
+
+
+  self.register = function($scope, $optionElement, $optionAttrs, interpolateValueFn, interpolateTextFn) {
```

Comment:
Yes, I meant `registerOption` :-)

So in the Angular code, elements that are wrapped in jqLite are prefixed with a dollar, and I always prefer to do the same with the attributes and scope, since they are "Angular" things, not from the user's application space. I guess that this is not that common so I don't mind dropping them if they would cause confusion.